deploy-as-code/helm/charts/core-services/egov-mdms-service/values.yaml

```diff
@@ -13,7 +13,7 @@ ingress:
 initContainers:
   gitSync:
     enabled: true
-    repo: "git@github.com:egovernments/egov-mdms-data"
+    repo: "git@github.com:misdwss/mdms-mgramseva"
```

Comment:
You do not need to modify helm chart. You can override this property in env file like [this](https://github.com/egovernments/DIGIT-DevOps/blob/master/deploy-as-code/helm/environments/dev.yaml#L226)